drivers/input/Kconfig.evdev

```diff
@@ -0,0 +1,11 @@
+# Copyright 2023 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config LINUX_EVDEV
```

Comment:
Should this be `config NATIVE_LINUX_EVDEV`?

Reply:
yeah good call, changed